MiniProject1.py

```diff
@@ -0,0 +1,96 @@
+# Kimber's file
+from pattern.en import *
+rhymeWords=dict() #dictionary of words that rhyme
+generalWords=dict() #markov chain dictionary
+#fo = open("ShakespeareSonnets.txt", "r")
+with open('ShakespeareSonnets.txt', 'r') as f:
+	read_data = f.read()
+f.closed
+sonnetz= read_data.split('\n\n')
+sonnetLine=[0, 1, 2, 3,4, 5,6 ,7, 8, 9, 10, 11, 12, 13]
+import random
+
+for k in range(0,len(sonnetz)-1):
+	lines=sonnetz[k].split('\n') #splits into lines
+
+	def DictRhymes():
+		getRhyme(0, 2) #English sonnet rhyme scheme (lines that rhyme)
+		getRhyme(1, 3)
+		getRhyme(4, 6)
+		getRhyme(5, 7)
+		getRhyme(8,10)
+		getRhyme(9,11)
+
+
+	def getRhyme(num1, num2): #num1 and num2 inputs represent which lines are selected (rhyming lines)
+		lastword1=lines[num1].split(' ')[-1] #takes the last word of every other line
+		lastword2=lines[num2].split(' ')[-1]
+		if(sentiment(lastword1)[0]<=.25 and sentiment(lastword2)[0]<=.25):
```

Comment:
Also in the case of the .25, having it as a variable at the top of your file or something would be really useful, so if you ever wanted to change it to .5 or something, you only have to change one variable, instead of the three times you use that number